catalog-rest-service/src/main/resources/json/schema/entity/services/connections/database/snowflakeConnection.json

```diff
@@ -66,8 +66,7 @@
     "queryTag": {
       "title": "Query Tag",
       "description": "Session query tag used to monitor usage on snoflake",
```

Comment:
We might need to add a one-liner explaining the necessary permissions to properly run the tagging